compose/ui/ui/api/desktop/ui.api

```diff
@@ -474,7 +474,6 @@ public final class androidx/compose/ui/awt/ComposePanel : javax/swing/JLayeredPa
 	public fun isFocusOwner ()Z
 	public fun isFocusable ()Z
 	public fun isRequestFocusEnabled ()Z
-	public fun remove (Ljava/awt/Component;)V
```

Comment:
It was an override, and it's still defined in super class. So it's not an API change.